### DISCUSSION TITLE
How to provide translated locals for the editor module in root module?

### DISCUSSION THREAD

**Discussion by exitlol** [sibiraj-s/ngx-editor#397] [Q&A] [answered]
I know DI factories and whatnot but i am puzzled how can I provide the already translated texts from my ngx-translate service into the module constructor?

Can someone help me with this?

**Comment by sibiraj-s** [comment, marked as answer]
This should be addressed by https://github.com/sibiraj-s/ngx-editor/pull/398 and published in [v12.4.0](https://github.com/sibiraj-s/ngx-editor/releases/tag/v12.4.0)

**Reply by sibiraj-s** [reply]
Thanks @ludoviccarretti for fixing this.

[a comment by an automated account is not shown]
